src/main/scala/org/dia/core/SRDD.scala

```diff
@@ -18,29 +18,25 @@ package org.dia.core
 
 import org.apache.spark._
 import org.apache.spark.rdd.RDD
-import org.dia.tensors.{AbstractTensor, TensorFactory}
-
+import org.dia.tensors.{ AbstractTensor, TensorFactory }
 import scala.collection.mutable
-
-//import scala.collection.immutable.HashMap
-//import scala.collection.parallel.mutable
 import scala.reflect.ClassTag
 
 /**
- * Scientific RDD. There are few differences between an sRDD and a normal RDD.
+ * Scientific RDD. There are few differences between an SRDD and a normal RDD.
```

Comment:
I don't get changing sRDD.scala to SRDD.scala, why? Since the proposal I've referred to it as the sRDD (aka "scientific" RDD). Why capitalize it?